apps/rehabstod/src/pages/CurrentSickLeaves/utils/getOccupationPlaceholder.ts

```diff
@@ -0,0 +1,14 @@
+import { OccupationType } from '../../../schemas/sickLeaveSchema'
+
+export const getOccupationPlaceholder = (selected: string[], occupationTypes: OccupationType[]) => {
```

Comment:
This is the same as the rekoStatus placeholder right? Maybe we could merge these two since the type objects look the same we could send in types: OccupationType[] | RekoStatusType[]

Reply:
Another option is that they take the same object for example TextIdFIlterType or something similar, as long as the id and name inside the object have the same name it will work against the backend

Reply:
From what I could tell there is a difference in that when no options are selected, occuaptionPlacerholder should be "Alla valda". Maybe this is also true for Reko? in that case I totally agree!